strings/src/main/res/values/strings.xml

```diff
@@ -499,6 +499,7 @@
     <string name="track_location">Track location</string>
     <string name="background_location_disabled">This form wants to track your location but tracking is disabled. Please enable in the %1$s menu above.</string>
     <string name="background_location_enabled">This form tracks your location. You can disable tracking in the %1$s menu above.</string>
+    <string name="validate">Check for errors</string>
```

Comment:
Could you add translator notes for this, please (HTML comment above)? Something like "Text of an option available when filling out a form. Tapping this option checks the form for constraint or required violations and navigates the user back to the first field with an error."

It can be really hard to translate without context. What's a check? What's an error? Languages may have many words for each!